big_scape/main.py

```diff
@@ -339,10 +341,15 @@ def align_callback(tasks_done: int):
     # prepare output files
     legacy_prepare_output(run["output_dir"], pfam_info)
 
+    # write full network file
+    write_full_network_file(run, all_bgc_records)
+
     # prepare output files per cutoff
     for cutoff in run["gcf_cutoffs"]:
-        # TODO: update to use records and not gbk regions
+        # TODO: update to use records and not gbk regions?
```

Comment:
TODO can be removed, I think

Reply:
legacy_prepare_cutoff_output()  still uses gbks, not records?